### PR TITLE
DP-20722: Modify image section on info page to allow text wrapping

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-20722

**To Test:**
- [ ] Add steps to test this feature

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

**Comment by tom-fleming** [comment]
@arthurbaghdas I have one comment about a class in the Mayflower pull request that we think is affecting tests. Also, we will want to resolve the merge conflict with composer.lock in this PR before merging.

File: conf/drupal/config/acquia_connector.settings.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ spi:
   site_name: REDACTED
   site_machine_name: REDACTED
   site_uuid: REDACTED
-  site_environment: prod
+  site_environment: null
```
Comment (tom-fleming) [inline conf/drupal/config/acquia_connector.settings.yml]: @arthurbaghdas Any idea why this value changed?

Comment (arthurbaghdas) [reply conf/drupal/config/acquia_connector.settings.yml]: Wrong export. Reverted

**Review by tom-fleming — APPROVED** [review]
LGTM, but we might want to hold on merging until Mayflower is updated. I will note this in the ticket as well.

File: conf/drupal/config/field.field.paragraph.image.field_image_administrative_title.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+dependencies:
+  config:
+    - field.storage.paragraph.field_image_administrative_title
+    - paragraphs.paragraphs_type.image
+id: paragraph.image.field_image_administrative_title
+field_name: field_image_administrative_title
+entity_type: paragraph
+bundle: image
+label: 'Administrative title'
+description: 'Title or brief description of the visualization. This isn''t printed on the page for sighted users but it''s used by visitors who use screen readers so they can understand what the visualization is about.'
```
Comment (tom-fleming) [inline conf/drupal/config/field.field.paragraph.image.field_image_administrative_title.yml]:
```suggestion
description: 'Title or brief description of the image. This isn''t printed on the page for sighted users but it''s used by visitors who use screen readers so they can understand what the image is about.'
```

File: conf/drupal/config/field.field.paragraph.image.field_image_caption.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    campaign_landing_basic: '0'
+    campaign_landing_restricted: '0'
+    rules_of_court_section: '0'
+    plain_text: '0'
+id: paragraph.image.field_image_caption
+field_name: field_image_caption
+entity_type: paragraph
+bundle: image
+label: Caption
+description: 'The caption will appear below the visualization. Limit 500 characters.'
```
Comment (tom-fleming) [inline conf/drupal/config/field.field.paragraph.image.field_image_caption.yml]:
```suggestion
description: 'The caption will appear below the image. Limit 500 characters.'
```

**Pull request merged**